src/integrationTest/java/uk/gov/hmcts/darts/ws/AddAudioWebServiceTest.java

```diff
@@ -1,24 +1,43 @@
 package uk.gov.hmcts.darts.ws;
 
 import com.service.mojdarts.synapps.com.AddAudioResponse;
-import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ArgumentsSource;
-import org.springframework.ws.soap.client.SoapFaultClientException;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.darts.addaudio.validator.AddAudioValidator;
+import uk.gov.hmcts.darts.common.multipart.XmlWithFileMultiPartRequest;
+import uk.gov.hmcts.darts.common.multipart.XmlWithFileMultiPartRequestHolder;
 import uk.gov.hmcts.darts.utils.IntegrationBase;
 import uk.gov.hmcts.darts.utils.TestUtils;
 import uk.gov.hmcts.darts.utils.client.SoapAssertionUtil;
 import uk.gov.hmcts.darts.utils.client.darts.DartsClientProvider;
 import uk.gov.hmcts.darts.utils.client.darts.DartsGatewayClient;
+import uk.gov.hmcts.darts.utils.matcher.MultipartDartsProxyContentPattern;
+import uk.gov.hmcts.darts.utils.multipart.DummyXmlWithFileMultiPartRequest;
+import uk.gov.hmcts.darts.workflow.command.AddAudioMidTierCommand;
 
+import java.util.Optional;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
 import static com.github.tomakehurst.wiremock.client.WireMock.ok;
 import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
 import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
 import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.verify;
 
 
 @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
-class AddAudioWebServiceTest  extends IntegrationBase {
+class AddAudioWebServiceTest extends IntegrationBase {
```

Comment:
Now that we've updated the README - I am wondering if you have an example for an addAudio request - would you be able to update `README-DARTSService-soapui-project.xml` to help the next person understand this endpoint to try it out?

Reply:
i will try and support this. The integration tests do send a multipart file using the mid tier library. I will try and capture this in SOAP UI

Reply:
This has now been performed and the README updated